packages/open-next/src/build.ts

```diff
@@ -523,8 +523,12 @@ async function createCacheAssets(monorepoRoot: string) {
     () => true,
     (filepath) => {
       const ext = path.extname(filepath);
-      const newFilePath =
+      let newFilePath =
         ext !== "" ? filepath.replace(ext, ".cache") : `${filepath}.cache`;
+      // Handle prefetch cache files for partial prerendering
+      if (newFilePath.endsWith(".prefetch.cache")) {
+        newFilePath = newFilePath.replace(".prefetch.cache", ".cache");
```

Comment:
This is very unlikely, but is it possible someone can have a `/apps/.prefetch/page.tsx` route. This would cause the actual route path to get replaced. Probably doesn't need to be handled but just thought I'd bring up the possibility 😄

Reply:
I had this exact same thought and came to the same conclusion as well. No need to worry about that i think.

And if they use PPR on that route, it would work